lib/Frontend/CompilerInvocation.cpp

```diff
@@ -238,6 +238,15 @@ static void SaveModuleInterfaceArgs(ModuleInterfaceOptions &Opts,
     return;
   ArgStringList RenderedArgs;
   for (auto A : Args) {
+    // SWIFT_ENABLE_TENSORFLOW: Copy
+    // `-Xllvm -enable-experimental-cross-file-derivative-registration` into the
+    // .swiftinterface file.
+    if (A->getOption().matches(options::OPT_Xllvm) &&
```

Comment:
Why is this `-Xllvm`?  Should this not be `-Xswift`?  This is part of the Swift side of things

Reply:
Basically all SILOptimizer flags are declared via `llvm::cl::opt<T>`, which adds a new `-Xllvm`-prefixed flag.

I actually haven't heard of `-Xswift`. Is there a way to register command line options prefixed with `-Xswift` instead of `-Xllvm`?

Reply:
I think you meant just a flag to `swiftc`? Yes, I think that would be better.